readme.config.js

```diff
@@ -7,8 +7,12 @@
                         "WithPicture":true,
                         "WithContributions":true,
                         "WithName":true,
+                        "WithEmail":true,
                         "WithLocation":true,
+                        "WithTwitterUsername":true,
+                        "WithNumberOfRepos":true
                       },
                       "TableDesign":"CELL_DESIGN",
-    }
+    },
+    "AppDescription": "",
```

Comment:
You don't need to put initial value inside the config.js... if you want to change anything you can do it here...
just make sure to add AppDescription inside the table I made for descriptions for config options

Reply:
Which file would that table be in? @sahandfarahani